extensions/Worker.Extensions.EventGrid/src/TypeConverters/EventGridStringArrayConverter.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.EventGrid.TypeConverters
+{
+    /// <summary>
+    /// Converter to bind to string[] parameter.
+    /// </summary>
+    [SupportedConverterType(typeof(string[]))]
+    internal class EventGridStringArrayConverter : IInputConverter
```

Comment:
This is already handled in the extension - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Config/EventGridExtensionConfigProvider.cs#L74-L75

Why do we want to implement it here as well?

Reply:
If it's handled in the extension we don't need to repeat - can probably remove the string and string[] converters 

@aishwarya can you validate that string bindings work without these new converters?

Reply:
Synced offline and came to the conclusion to keep the StringArrayConverter, since the JSONConverter doesnt' work as expected